docs/account-management/auth/api-key-auth.md

```diff
@@ -0,0 +1,130 @@
+---
+title: API Key Authentication
+sidebar_position: 10
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+API keys are the recommended authentication method for access by apps and APIs.
```

Comment:
```suggestion
API keys are the recommended authentication method for access by applications and APIs.
```

Probably better to consistently spell out "application" instead of short-handing it to "apps."

@diana-macrometa Any thoughts?